client/modules/IDE/components/SketchList.jsx

```diff
@@ -194,6 +212,16 @@ class SketchListRowBase extends React.Component {
           <td>{formatDateCell(sketch.updatedAt, mobile)}</td>
           {this.renderDropdown()}
         </tr>
+        {this.state.preview ? (
+          <ExamplePreview
+            url={`/editor/${this.props.username}/projects/${slugify(
+              this.props.sketch.name,
+              '_'
+            )}`}
+          />
+        ) : (
+          <div>loading</div>
```

Comment:
If `this.state.preview` is `false` then we would show nothing.  Not a loader.  You can do `{this.state.preview && (` or keep the ternary and use `) : null}` for the `false` case.